runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/Engine.java

```diff
@@ -193,6 +177,7 @@ public final class Engine implements Collector, AutoCloseable
         final Map<String, Guard> guardsByType = guards.stream()
             .collect(Collectors.toMap(g -> g.name(), g -> g));
 
+        this.configPath = config.configPath();
```

Comment:
Looks like `configPath` is no longer needed as `Engine` field after `resovlePath(...)` method is moved to `EngineWorker` implementation directly.